setup.cfg

```diff
@@ -8,11 +8,9 @@ force_grid_wrap=0
 use_parentheses=True
 line_length=88
 indent = "    "
-# by default isort don't check module indexes
-not_skip = __init__.py
```

Comment:
Not needed with isort 5.x

Reply:
Can we instead use the new profile option in isort? 
 https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

Reply:
> Can we instead use the new profile option in isort? https://pycqa.github.io/isort/docs/configuration/black_compatibility.html#using-a-config-file-such-as-isortcfg

Done